src/Ruleset.php

```diff
@@ -461,21 +464,22 @@ public function showSniffDeprecations()
 
         $summaryLine = wordwrap($summaryLine, $reportWidth, PHP_EOL);
         if ($this->config->colors === true) {
-            echo "\033[33m".$summaryLine."\033[0m".PHP_EOL;
+            StatusWriter::write("\033[33m".$summaryLine."\033[0m");
         } else {
-            echo $summaryLine.PHP_EOL;
+            StatusWriter::write($summaryLine);
         }
 
         $messages = implode(PHP_EOL, $messages);
         if ($this->config->colors === false) {
             $messages = Common::stripColors($messages);
         }
 
-        echo str_repeat('-', min(($maxActualWidth + 4), $reportWidth)).PHP_EOL;
-        echo $messages;
+        StatusWriter::write(str_repeat('-', min(($maxActualWidth + 4), $reportWidth)));
+        StatusWriter::write($messages, 0, 0);
 
         $closer = wordwrap('Deprecated sniffs are still run, but will stop working at some point in the future.', $reportWidth, PHP_EOL);
-        echo PHP_EOL.PHP_EOL.$closer.PHP_EOL.PHP_EOL;
+        StatusWriter::write(PHP_EOL);
```

Comment:
I find it counterintuitive that `StatusWriter::write(PHP_EOL);` is used to output two newlines instead of one. The same goes for `StatusWriter::writer('')` to output one newline.

Since this pattern has been used quite a few times in this PR, I wonder if it would make sense to create a helper method to output newlines. The name of the method should help clarify what is happening.

If not, in this particular case, I'm inclined to think that `StatusWriter::write('', 0, 2);` is better than `StatusWriter::write(PHP_EOL);` as someone reading the code will have to check the parameters and then it will be clear that two newlines will be printed.

Reply:
Good suggestion. Added.